src/app/dataset-view/dataset-view-menu/dataset-view-menu.component.html

```diff
@@ -106,60 +106,291 @@
+                            <div class="select-menu-title text-bold d-flex align-item-center d-flex align-items-center">
+                                Protocols
                             </div>
-                            <button
-                                class="select-menu-close-btn"
-                                type="button"
-                                aria-label="Close menu"
-                                data-action="click:notifications-list-subscription-form#closeMenu"
-                            >
-                                <svg-icon name="close"></svg-icon>
-                            </button>
+                            <a class="fs-12">Manage API tokens</a>
                         </header>
                         <div
                             class="select-menu-list"
                             (click)="$event.stopPropagation()"
                             (keydown)="$event.stopPropagation()"
                         >
-                            <mat-tab-group>
-                                <mat-tab label="Download">
+                            <mat-tab-group [animationDuration]="0">
+                                <mat-tab>
+                                    <ng-template mat-tab-label>
+                                        <div class="d-flex align-items-center">
+                                            <mat-icon class="text-muted fs-5 mt-2">link</mat-icon>
+                                            Link
+                                        </div>
+                                    </ng-template>
                                     <div class="select-menu-item flex-column">
-                                        <div
-                                            class="select-menu-item align-items-start p-0 border-0"
-                                            (click)="$event.stopPropagation()"
-                                            (keydown)="$event.stopPropagation()"
-                                        >
-                                            <p>Using kamu-cli:</p>
-                                        </div>
-                                        <div
-                                            class="select-menu-item align-items-start p-0 border-0"
-                                            (click)="$event.stopPropagation()"
-                                            (keydown)="$event.stopPropagation()"
-                                        >
+                                        <div class="input-group mb-1">
+                                            <span class="fs-12 text-muted"> Share this dataset on the Web:</span>
+                                        </div>
+
+                                        <div class="select-menu-item align-items-start p-0 border-0 mb-2">
+                                            <div class="input-group">
+                                                <input
```

Comment:
I can currently edit these URLs - they should be made read-only.

Reply:
Fixed